go.mod

```diff
@@ -3,14 +3,14 @@ module oras.land/oras-go
 go 1.18
 
 require (
-	github.com/containerd/containerd v1.7.0
+	github.com/containerd/containerd v1.6.19
```

Comment:
Hmm, I wonder why the downgrade

Reply:
I think this is because `containerd v1.7.0` referenced `image-spec` via a commit instead of a tag.